ingest/validation/validate_metadata.py

```diff
@@ -1345,10 +1423,20 @@ def detect_excel_drag(metadata, convention):
                         )
                         if multiply_assigned:
                             msg += f"Check for mismatches between ontology ID and provided ontology label(s) {multiply_assigned}\n"
-                    metadata.store_validation_issue("error", "ontology", msg)
+                    metadata.store_validation_issue(
+                        "error",
+                        "ontology",
+                        msg,
+                        issue_name="content:metadata:mismatched-id-label",
+                    )
                     dev_logger.exception(msg)
             except ValueError as valueError:
```

Comment:
```suggestion
            except ValueError as value_error:
```

Reply:
Oops, missed this suggestion - will include as part of the expression matrix follow-on ticket.